### DISCUSSION TITLE
Disable Print and Download icons in the Toolbar

### DISCUSSION THREAD

**Discussion by VamsiBoggala** [mozilla/pdf.js#18181] [Q&A]
![Screenshot (9)](https://github.com/mozilla/pdf.js/assets/89140100/856b633e-94a3-45cc-a937-e8a783bf2284)

"I am trying to disable the download and print functionalities through the URL 'https://mozilla.github.io/pdf.js/web/viewer.html?file=$url&print=false&download=false&secondaryDownload=false,' but I am not able to hide those buttons. If possible, please let me know how to do it."

For reference please find the above url.

**Comment by mighty-mayank** [comment]
hey !
have you found a workaround yet ?
i am also stuck with the same problem

**Comment by katcaola** [comment]
I made feature request #20141 for a read only / view only toolbar which requests to remove the print/download buttons and other buttons for editing. It may be useful to `+1` / follow that feature request thread. Someone asked to be assigned to the task so maybe it will happen? 👍

**Reply by GeneriicName** [reply]
You can simply remove them from the html and their corresponding js from the viewer.mjs

I removed print and presentation for example.
